x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/elasticsearch/ElasticsearchInternalService.java

```diff
@@ -904,12 +917,19 @@ private List<Model> defaultConfigs(boolean useLinuxOptimizedModel) {
             ),
             ChunkingSettingsBuilder.DEFAULT_SETTINGS
         );
-        return List.of(defaultElser, defaultE5);
+        var defaultRerank = new ElasticRerankerModel(
```

Comment:
Just checking that this should be the model we use and not `CustomElandRerankModel`